.github/workflows/main.yml

```diff
@@ -32,3 +32,8 @@ jobs:
         run: make
         env:
           MSP430_TOOLCHAIN_PATH: /opt/msp430-gcc
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: usb-uart-bridge
+          path: usb_uart_bridge*.*
```

Comment:
Can change to `usb_uart_bridge.*` unless you want to capture the build files for the launchpad